src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,53 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int buy = 0;
+        int supply = 0;
+        int result = 0;
+
+        try (BufferedReader reader =
+                     new BufferedReader(new FileReader(fromFileName))) { //try-with-resources
+
+            String line;
+
+            while ((line = reader.readLine()) != null) { // Читаємо файл, рядок за рядком
+
+                String[] parts = line.split(",");
+                String definition = parts[0];
+                int amountOfDefinition = Integer.parseInt(parts[1]);
+                if (definition.equals("buy")) {
+                    buy += amountOfDefinition;
+                }
+                if (definition.equals("supply")) {
+                    supply += amountOfDefinition;
+                }
```

Comment:
Consider using `else if` for the second condition to avoid unnecessary checks. If the first condition is true, there's no need to check the second condition.